### PR TITLE
Add manual set of ZeroMQ_INCLUDE_DIR

### PR DESCRIPTION
**Pull request by Purg**
To go along with "manual" static library discovery.

### CODE REVIEW

**Review by mleotta — CHANGES_REQUESTED** [review]
Please target the release branch for bug fixes

File: doc/release-notes/master.txt [hunk 1]
```diff
@@ -12,3 +12,11 @@ Build System
...
 * Removed some CMake statements and support for old CMake version predating the
   current minimum version specification.
+
+
+Bug Fixes
+---------
+
+Build System
+
+* Fix mizzing `ZeroMQ_INCLUDE_DIR` set when building statically.
```
Comment (dstoup) [inline doc/release-notes/master.txt:22]: spelling error ... 'mizzing'

Comment (Purg) [reply doc/release-notes/master.txt:22]: Oh my. Embarrassing. Will fix with base transition.

**Review by tao558 — APPROVED** [review]

**Comment by Purg** [comment]
Checking back in. This is now based on release and has a positive review from @tao558. What's the next action here / who's responsible for merging?

**Pull request merged**